### DISCUSSION TITLE
Regarding the included theme

### DISCUSSION THREAD

**Discussion by akvariefisk** [BoidCMS/BoidCMS#18] [Ideas]
Hello, 
The themes is using Google fonts from Google, not running on a local storage. 

I read about a court "Sued by Using Google Fonts: Data Privacy and GDPR" in Munich.
https://wicki.io/posts/2022-02-goodbye-google-fonts-data-privacy-gdpr/

How about instead Self-Host Google Fonts included in the theme to avoid this matter out of the box. 

Just a thougt. 
Erik

**Comment by sysadev** [comment]
Hum... That's very interesting. Thanks for sharing
